controller/space.go

```diff
@@ -53,6 +54,26 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
+		/*
```

Comment:
@sbose78 Code should be self-explanatory but if not, you can keep it ;). I saw many code commented in the code. Not a blocker, just go ahead.